tools/src/test/java/org/apache/pdfbox/tools/imageio/TestAddImage.java

```diff
@@ -0,0 +1,45 @@
+package org.apache.pdfbox.tools.imageio;
+import java.io.File;
+import java.io.IOException;
+import java.io.StringReader;
+
+import junit.framework.TestCase;
+import junit.framework.TestSuite;
+
+import org.apache.pdfbox.pdmodel.PDDocument;
+import org.apache.pdfbox.pdmodel.PDPageContentStream;
+import org.apache.pdfbox.pdmodel.graphics.image.PDImageXObject;
+import org.apache.pdfbox.tools.AddImage;
+import org.junit.Test;
+import static org.junit.Assert.assertEquals;
+
+public class TestAddImage extends TestCase {
+
+@Test
+public void testLoadFileAndInitializeStream() throws IOException {
+
+    AddImage add = new AddImage();
+    String expected = "C:/Users/Asher/Downloads/itext7start/itext7start/mavenproject2/results/chapter01/hello.pdf";
```

Comment:
Can that be changed based on the user? Im assuming that is the location of PDF file created.

Reply:
Will that fail if ran on another PC?

Reply:
That should be an outdated version, the latest version has a path to somewhere within the project directory

Reply:
If you can find the commit I made dated today (since I made it after midnight), that should have the up to date code